install.ps1

```diff
@@ -86,6 +86,11 @@ function Move-ConfigFolder {
 
 #region Main
 if ($PSVersionTable.PSVersion.Major -ge $PSMinVersion) {
+  if ([bool]([Security.Principal.WindowsIdentity]::GetCurrent().Groups -contains 'S-1-5-32-544')) {
+    Write-Log -ActionText "The script was ran as Administrator. Exiting..." | Out-Null
```

Comment:
We should probably add another line telling users NOT to do that instead of leaving them confused imo.